boilerplates/aws-lambda-serverless/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@batijs/aws-lambda-serverless",
+  "private": true,
+  "version": "0.0.1",
+  "description": "",
+  "type": "module",
+  "scripts": {
+    "check-types": "tsc --noEmit",
+    "build": "bati-compile-boilerplate"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "MIT",
+  "devDependencies": {
+    "@batijs/compile": "workspace:*",
+    "@types/node": "^18.19.45",
+    "esbuild-plugin-copy": "^2.1.1",
+    "serverless": "^3.39.0",
+    "serverless-cloudfront-invalidate": "^1.12.2",
+    "serverless-esbuild": "^1.52.1",
+    "serverless-s3-sync": "^3.3.0",
+    "serverless-scriptable-plugin": "^1.3.1"
+  },
+  "dependencies": {
+    "@batijs/core": "workspace:*"
+  },
+  "files": [
+    "dist/"
+  ],
+  "bati": {
+    "if": {
+      "flag": {
+        "$all": [
+          "hono",
+          "aws-lambda-serverless"
+        ]
```

Comment:
TODO: either support all servers (probably not possible), or add a [rule](https://github.com/vikejs/bati/blob/main/packages/features/src/rules/rules.ts).

Reply:
added support for `hattip` server, extended the rules and error messages - express would be possible too, but the current implementation does not export the app alone without the local server.

Reply:
:thinking: Would this also work with `h3` and `fastify`? If yes, we should support all of them. If not, let's stick to `hattip` and `hono`.

Reply:
> 🤔 Would this also work with `h3` and `fastify`? If yes, we should support all of them. If not, let's stick to `hattip` and `hono`.

Fastify, Express,h3 need to be rewritten similar to hono into a file which only exports the server (`export default app`) and a file similar to `hono-entry.node.ts` with the part which starts listening to a port. Most of them can than be wrapped with [serverless-http](https://github.com/dougmoscrop/serverless-http)

My current priorities are to fully support one server framework `hono` until I am finished with all other integrations (e.g. sentry).

Reply:
> My current priorities are to fully support one server framework hono until I am finished with all other integrations (e.g. sentry).

Fair enough :+1: